WooCommerce/Classes/POS/Presentation/PointOfSaleCollectCashView.swift

```diff
@@ -26,7 +32,7 @@ struct PointOfSaleCollectCashView: View {
 
     var body: some View {
         ScrollView {
-            VStack(alignment: .center, spacing: conditionalPadding(13)) {
+            VStack(alignment: .center, spacing: conditionalPadding(8)) {
```

Comment:
Final spacing can be configured in the spacing/padding task. For now I just made them the same in both cash and email views